resources/fixtures/Martin/Martin-MAC250-Entour.qxf

```diff
@@ -61,7 +61,7 @@
   <Capability Min="132" Max="132" Preset="ColorMacro" Res1="#9c39b5">Purple 502</Capability>
   <Capability Min="133" Max="142" Preset="ColorDoubleMacro" Res1="#9c39b5" Res2="#ffffff">White</Capability>
   <Capability Min="143" Max="143" Preset="ColorMacro" Res1="#ffffff">White</Capability>
-  <Capability Min="144" Max="155" Preset="ColorDoubleMacro" Res1="" Res2="#ffffff">/White</Capability>
+  <Capability Min="144" Max="155" Preset="ColorMacro" Res1="#ffffff">/White</Capability>
```

Comment:
This should look like the next modification, but please check the manual. One range is for continuous scroll, while the other is for stepped scroll

Reply:
Yes, I will clarify that. There is also another error in the "Dark Green / White" which should be "Dark Green / Purple" in the name (the codes are already good).
It is still strange as the continuous rotation position "0" is White and the Continuous rotation position 143 is also White, turning continuously to the next White slot between 144 and 155 (which turns to the White at position 0).